nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Utilities/JsonHelperTests.cs

```diff
@@ -9,13 +10,20 @@ namespace NuGetUpdater.Core.Test.Utilities;
 
 public class JsonHelperTests
 {
+    private static readonly JsonDocumentOptions DocumentOptions = new()
+    {
+        CommentHandling = JsonCommentHandling.Skip,
+    };
+
     [Theory]
     [MemberData(nameof(JsonUpdaterTestData))]
     public void UpdateJsonPreservingComments(string json, string[] propertyPath, string newValue, string expectedJson)
     {
         var updatedJson = JsonHelper.UpdateJsonProperty(json, propertyPath, newValue, StringComparison.OrdinalIgnoreCase).Replace("\r", string.Empty);
         expectedJson = expectedJson.Replace("\r", string.Empty);
         Assert.Equal(expectedJson, updatedJson);
+        using var document = JsonDocument.Parse(updatedJson, DocumentOptions);
+        Assert.Equal(JsonValueKind.Object, document.RootElement.ValueKind);
```

Comment:
This assert fails for the additional test case without the fix as the string fails to parse as valid JSON.